mssql-plugin/src/main/java/io/cdap/plugin/mssql/SqlServerSourceDBRecord.java

```diff
@@ -45,6 +47,17 @@ public SqlServerSourceDBRecord() {
   protected void handleField(ResultSet resultSet, StructuredRecord.Builder recordBuilder,
       Schema.Field field,
       int columnIndex, int sqlType, int sqlPrecision, int sqlScale) throws SQLException {
+    if (SqlServerSourceSchemaReader.shouldConvertToDatetime(resultSet.getMetaData(), columnIndex)) {
+      try {
+        Method getLocalDateTime = resultSet.getClass().getMethod("getDateTime", int.class);
+        recordBuilder.setDateTime(field.getName(),
+                                  ((Timestamp) getLocalDateTime.invoke(resultSet, columnIndex)).toLocalDateTime());
+        return;
+      } catch (Exception e) {
```

Comment:
can this be the specific Exception the method can throw

Reply:
done :)